docs/faqs-and-tips.md

```diff
@@ -0,0 +1,21 @@
+---
+title: FAQs and Tips
+description: Learn about top tips from and common problems faced by the React Native Firebase community.
+next: /releases
+previous: /migrating-to-v6
+---
+
+Over the years, there’s been a lot of discussions on our [GitHub]( https://github.com/invertase/react-native-firebase) and [Discord]( https://invertase.link/discord). Many of them have been about common problems developers face when using our package, and some of them resulted in very good advice being given.
+
+In order to save others time and frustration, this page has been created to document some of these common problems and good pieces of advice.
+If you come across a discussion that results in great advice that can benefit many developers, or a discussion that resolves a problem that many developers encounter, please do add it here! Someone will definitely be grateful.
+
+
+# FAQs
+
+Here are some commonly encountered problems:
+
+[TODO]
+
+# Tips
+ - Advice on cleanly creating a test and production environment for your React Native Firebase App: [#3504]( https://github.com/invertase/react-native-firebase/issues/3504)
```

Comment:
```suggestion
 - Advice on supporting multiple environments (for example, dev, prod, maybe also staging, qa) for your React Native Firebase App: [#3504]( https://github.com/invertase/react-native-firebase/issues/3504)
```

Reply:
Meant to provide some text as well - I love this initiative, and just for that specific one I wanted to give more context leading up to the link. But just getting it going is great.

Another FAQ that always comes up @Ehesp / @Salakar are the classes of errors that lead to the answer 'run `npx react-native-clean-project && cd ios && pod install` - like differences between debug and release on iOS, version mismatch of packages